python/pyarrow/table.pxi

```diff
@@ -495,8 +495,8 @@ cdef class RecordBatch(_PandasConvertible):
     """
     Batch of rows of columns of equal length
 
-    Warning
-    -------
+    Warns
```

Comment:
No. "Warns" is for Python warnings raised by the function. This one is a documentation-level warning, so should revert to the original spelling.

Reply:
Fixed.